### ISSUE TITLE
FR: Adding margins when reading comics

### ISSUE THREAD

**Issue by pawanbasnet** [koreader/koreader#11836]
**Does your feature request involve difficulty completing a task? Please describe.**
I read my comics on a desktop using monitor. Currently using fit height makes the page size really small to read and fit width make each page really stretched out across entire monitor width. Having a option to add margin to each side of comic reader much like book reader would help medicate the issue so it gives users more control of  comic reader size

**Describe the solution you'd like**
I believe having a margin option would help resize the display for comics pages so they are not super stretched out across entire monitor.

**Describe alternatives you've considered**
Alternatives that I am currently using is opening Kavita in its own window and resizing the Firefox window to the size that comic book is ideal for reading. If I am binge reading for longer period, I also edit the CSS file to increase the margin size for the page. It does revert again if I refresh or back out of reading though.

**Additional context**
Add any additional context or screenshots about the feature request here.
![Screenshot 2024-05-16 182201](https://github.com/koreader/koreader/assets/28436581/26d5387e-b995-4e89-92a5-7019ecd0977f)

**Comment by offset-torque** [comment]
I am trying to understand this **pawanbasnet**:
If you are using KOReader on desktop, why don't you resize your window horizontally to a comfortable size?
Why your KOReader window has to cover your whole screen horizontally?

**Comment by Frenzie** [comment]
I can ima

**Comment by jonnyl2** [comment]
>But more to the point, the bottom menu already has a margin setting. ;-)

Not sure if there's anything different for comics/image-files, but on PDF/DJVU, that bottom menu margin setting can only _remove_ margins, but not add them. I also sometimes would like to add extra margins when reading scanned PDF/DJVU books. To achieve that, I have to use a zoom factor, e.g. 0.9, from the 3rd tab in the bottom menu instead.

**Comment by jonnyl2** [comment]
>fit to content width and then you add just a little margin

Yes, this works to gain back some of the removed margin, but if there wasn't any in the document to begin with, it won't add any extra margin.

**Comment by Frenzie** [comment]
Thanks for clarifying.
